plugins/woocommerce/includes/wc-rest-functions.php

```diff
@@ -457,3 +459,20 @@ function wc_rest_should_load_namespace( string $ns, string $rest_route = '' ): b
 	 */
 	return apply_filters( 'wc_rest_should_load_namespace', str_starts_with( $rest_route, $ns ), $ns, $rest_route, $known_namespaces );
 }
+
+/**
+ * Check if the WooCommerce REST API v4 feature is enabled.
+ *
+ * @return bool True if the REST API v4 feature is enabled, false otherwise.
+ */
+function wc_rest_api_v4_is_enabled(): bool {
+	// Container may not be initialized yet during early autoloading.
+	if ( ! function_exists( 'wc_get_container' ) || empty( $GLOBALS['wc_container'] ) ) {
+		return false;
+	}
+	$container = wc_get_container();
+	if ( ! is_object( $container ) ) {
+		return false;
+	}
+	return (bool) Features::is_enabled( 'rest-api-v4' );
```

Comment:
Can we use `Features::is_enabled()` directly instead of creating a permanent global function that has to be maintained?

Reply:
No action needed for this feedback! I fixed it in https://github.com/woocommerce/woocommerce/pull/60690

Reply:
Oups, GitHub UI was old when I posted this, just saw your comment @gigitux ! Thanks for fixing it!